spring-boot-autoconfigure/src/main/java/org/axonframework/springboot/autoconfig/InterceptorAutoConfiguration.java

```diff
@@ -34,8 +39,14 @@
 import java.util.Optional;
 
 /**
- * Interceptor autoconfiguration class for Axon Framework application. Discovers {@link MessageHandlerInterceptor}s and {@link MessageDispatchInterceptor}
- * and registers them with the respective buses and gateways.
+ * Interceptor autoconfiguration class for Axon Framework application. Discovers {@link MessageHandlerInterceptor}s and
```

Comment:
I received a question from a customer regarding that:
```
I just want to ask whether the [new auto-registering feature](https://github.com/AxonFramework/AxonFramework/pull/3203) will make our interceptors to be registered twice because we already have a similar mechanism in place that finds interceptor beans and registers them. If that's the case, we shouldn't upgrade to the new version of the framework.
```
Is he right that the interceptors will be duplicated?
Could you confirm @smcvb?
I think only advice we can have for such case it's to replace the mechanism with this one or exclude the `InterceptorAutoConfiguration` from Spring Boot context.

Reply:
It is hard to say with 100% certainty whether this will occur without knowing their implementation. But, I do think it's a safe bet.
And yes, I agree with your response: I would replace the custom code with Axon Framework's approach.
Not upgrading for that purpose, to me, sounds like a bad choice. Purely because we'll make patch release for 4.11, ideally not for 4.10 anymore.

Perhaps this user could take a peak and see if they have some additional features in that area that everybody benefit from. And if so, if he/she/they would want to make a PR for that. :-)

Reply:
Thanks! :)